src/test/java/org/testeditor/web/xtext/index/api/EObjectDescriptionSerializationTest.xtend

```diff
@@ -0,0 +1,80 @@
+package org.testeditor.web.xtext.index.api
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.module.SimpleModule
+import io.dropwizard.jackson.Jackson
+import org.eclipse.emf.ecore.EPackage
+import org.eclipse.emf.ecore.util.EcoreUtil
+import org.eclipse.xtext.XtextFactory
+import org.eclipse.xtext.XtextPackage
+import org.eclipse.xtext.resource.EObjectDescription
+import org.eclipse.xtext.resource.IEObjectDescription
+import org.junit.BeforeClass
+import org.junit.Test
+import org.testeditor.web.xtext.index.serialization.EObjectDescriptionDeserializer
+import org.testeditor.web.xtext.index.serialization.EObjectDescriptionSerializer
+
+import static org.assertj.core.api.Assertions.assertThat
+
+import static extension io.dropwizard.testing.FixtureHelpers.*
+
+class EObjectDescriptionSerializationTest {
+	static extension ObjectMapper mapper = Jackson.newObjectMapper
+
+	@BeforeClass
+	static def void ensureTestAssumptions() {
+		serializersHaveToBeRegistered()
+		metamodelMustBeKnown()
+	}
+
+	def static serializersHaveToBeRegistered() {
```

Comment:
name is misleading, how about registerSerializers

Reply:
Well, its about the *assumptions* the test has to make about the environment it is run in (global preconditions / "givens"). So if you read the method `ensureTestAssumptions`, it documents these assumptions pretty well (or so I thought). Anyways, your suggestions probably work just as well :)